api/hackfsu_com/settings.py

```diff
@@ -35,7 +35,8 @@
     '127.0.0.1',
     '[::1]',
     'hackfsu.com',
-    '.hackfsu.com'
+    '.hackfsu.com',
+	'192.168.99.100'
```

Comment:
There's something odd about the spacing here. Investigate whether you have tabs or spaces set in Pycharm.